src/NzbDrone.Core/Datastore/Migration/071_unknown_quality_in_profile.cs

```diff
@@ -149,6 +158,43 @@ public void SplitQualityAppend(int find, int quality)
             }
         }
 
+        public void UpdateQualityToQualityDefinition()
```

Comment:
Should we be updating a past migration? What does this do for us?

Reply:
This is leftover code from the old custom formats idea. Should be removed